mullvad-types/Cargo.toml

```diff
@@ -26,3 +26,6 @@ clap = { workspace = true , optional = true }
 
 [target.'cfg(target_os = "android")'.dependencies]
 jnix = { version = "0.5.1", features = ["derive"] }
+
+[dev-dependencies]
+proptest = { workspace = true }
```

Comment:
Is proptest used in mullvad-types anymore? `rg proptest mullvad-types/` doesn't find anything other than this toml file